build/contrib/builder/image/Dockerfile

```diff
@@ -1,48 +1,30 @@
 FROM centos:centos7 as builder
```

Comment:
Actually this Dockerfile was used by the quickstart using etcd configuration component ,see https://mosn.io/layotto/#/zh/start/configuration/start

If it is modified,the existing quickstart won't work.
So there are two options:
A. write a new Dockerfile instead of modifying this existing one
B. modify this one,but meanwhile update the etcd quickstart doc(e.g. ask users to start etcd container and layotto container respectively, or add a docker-compose file for that quickstart)

For simplicity,option A is much easier?